.gitattributes

```
@@ -0,0 +1,10 @@
+* text=auto
+
+.gitattributes export-ignore
+.gitignore export-ignore
+composer.lock export-ignore
+.travis.yml export-ignore
+/ci export-ignore
+/tests export-ignore
+phpunit.xml.dist export-ignore
```

The PHPUnit config is in there twice, was probably meant to add PHPCS instead.
```suggestion
phpcs.xml.dist export-ignore
```

---

Oops.